bitnami/moodle/templates/NOTES.txt

```diff
@@ -60,24 +60,18 @@ host. To configure Moodle to use and external database host:
 
 {{- end }}
 
-{{- include "moodle.checkRollingTags" . }}
+{{- include "common.warnings.rollingTag" .Values.image }}
+{{- include "common.warnings.rollingTag" .Values.metrics.image }}
+{{- $passwordValidationErrors := list -}}
 
-{{- $passwordErrors := list -}}
 {{- if not .Values.existingSecret -}}
-  {{- $requiredMoodlePassword := dict "valueKey" "moodlePassword" "secret" (include "moodle.fullname" .) "field" "moodle-password" "context" $ -}}
+  {{- $secretName := include "moodle.secretName" . -}}
+  {{- $requiredMoodlePassword := dict "valueKey" "moodlePassword" "secret" $secretName "field" "moodle-password" "context" $ -}}
   {{- $requiredMoodlePasswordError := include "common.validations.values.single.empty" $requiredMoodlePassword -}}
-  {{- $passwordErrors =  append $passwordErrors $requiredMoodlePasswordError -}}
+  {{- $passwordValidationErrors = append $passwordValidationErrors $requiredMoodlePasswordError -}}
 {{- end -}}
 
-{{- $passwordMysqlErrors := include "common.validations.values.mariadb.passwords" (dict "secretName" (include "moodle.mariadb.fullname" .) "context" $) -}}
-{{- $passwordErrors = append $passwordErrors $passwordMysqlErrors -}}
-
-{{- include "common.errors.upgrade.passwords.empty" (dict "validationErrors" $passwordErrors "context" $) -}}
-
-{{- if and .Values.externalDatabase.host .Release.IsUpgrade -}}
-  {{- $externaldbSecretName := printf "%s-%s" .Release.Name "externaldb" -}}
-  {{- $requiredExternalPassword := dict "valueKey" "externalDatabase.password" "secret" $externaldbSecretName "field" "db-password" }}
-
-WARNING: Review values for the following password in the command, if they are correct please ignore this notice.
-  {{- include "common.validations.values.multiple.empty" (dict "required" (list $requiredExternalPassword) "context" $) -}}
-{{- end -}}
```

Comment:
This is still valid.

Reply:
I don't know if we should add this after standardising the chart, what do you think?

Reply:
yes, you are right